src/main/java/spring/service/dto/UserResponseDto.java

```diff
@@ -0,0 +1,25 @@
+package spring.service.dto;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserResponseDto {
+    private String login;
+    private String password;
```

Comment:
probably you should reconsider adding user password to ResponseDto; 
Response dto should not contain it, it's unsecured